spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
```

Comment:
Done. I think I addressed most of your suggestions. I also used nested `describes`, I wasn't able to find out if that's a good practice in ruby, but it seems good to me. Let me know what you think. Also the search by `month` seems cheesy. I will open an issue about it.